spec/omnicache/store_spec.rb

```diff
@@ -292,4 +304,95 @@ def self.load(value)
       expect { store.delete("key") }.to change(store, :current_size_bytes).to(0)
     end
   end
+
+  context "when Datadog is available" do
```

Comment:
IMO, this shouldn't be its own context and instead each example should fall within the `describe`d methods above